Gemfile.lock

```diff
@@ -0,0 +1,155 @@
+GEM
```

Comment:
Gemfile.lock basically `lock` down versions of Gems. Don't worry this is auto-generated file
Run

```
bundle update
```

to generate this file.